tests/src/Interop/StructMarshalling/PInvoke/MarshalStructAsParamDLL.h

```diff
@@ -199,13 +169,14 @@ union InnerArrayExplicit // size = 32 bytes
 	{
 		LONG64 _unused0;
 		LPCSTR f4;
-	}; 
-
+	} s; 
 };
 
 
 #ifdef WINDOWS
 	#ifdef _WIN64
+        #pragma warning(push) 
+        #pragma warning(disable: 4201) // nonstandard extension used: nameless struct/union
```

Comment:
I suppose I could have gotten rid of this and then add a similar struct to the non-Windows version of OUTER3, but since this is already Windows/VC++ specific code I simply disabled the warning.